src/strategies_and_params/spectralstrategy.jl

```diff
@@ -18,11 +18,12 @@ num_spectral_states(::SpectralStrategy{S}) where {S} = S
     GramSchmidt{S} <: SpectralStrategy{S}
 
 Use the Gram-Schmidt procedure to orthogonalize the excited states. A total of `S` spectral
-states are used in the simulation.
+states are used in the simulation, and they are orthogonalised every `orthogonalisation_interval` steps.
 """
-struct GramSchmidt{S} <: SpectralStrategy{S} end
+struct GramSchmidt{S} <: SpectralStrategy{S}
+    orthogonalisation_interval::Int
+end
 
-function GramSchmidt(num_spectral_states = 1)
-    num_spectral_states > 1 && throw(ArgumentError("num_spectral_states > 1 is currently not supported."))
-    GramSchmidt{num_spectral_states}()
+function GramSchmidt(num_spectral_states = 1, orthogonalisation_interval = 1)
+    return GramSchmidt{num_spectral_states}(orthogonalisation_interval)
```

Comment:
```suggestion
function GramSchmidt(S = 1; orthogonalization_interval = 1)
    return GramSchmidt{S}(orthogonalization_interval)
```
I suggest making `orthogonalization_interval` a keyword argument. I hope this makes the role of the two arguments a bit less confusing (currently `GramSchmidt(3, 15) == GramSchmidt{3}(15)`. 

It would be great to add a method for `Base.show` that prints it as
`GramSchmidt(3; orthogonalization_interval=15)`

Also, we are trying to stick with American spelling for the code.

Reply:
Ok, I'll add a new `Base.show` method.